src/components/PostComment.js

```diff
@@ -0,0 +1,60 @@
+import React, {Component} from 'react'
+import { Button, Comment, Form, Header } from 'semantic-ui-react'
```

Comment:
I get compilation error. And why use semantic ui when you have material ui?

Reply:
they have some nice components that are not available with material